joomla/joomla-cms
# Restore Calendar.setup function for B/C

Pull Request for Issue https://github.com/joomla/joomla-cms/issues/15251 .

### Summary of Changes
Adds a proxy for Calendar.setup(object) to restore backwards compatibility.

### Testing Instructions
Apply patch.
Go to Administrator and edit an article
go to publishing tab
open browsers console and paste/run:
```js
        Calendar.setup({
            inputField: 'jform_publish_up',
            showOthers: '0'
        });

```

UPD: The showOthers can be `true` or `"true"` for enabled and anything else for disabled

the cal should reinitialise, to confirm open it again you should see the table without the days for prev/next month

### Expected result



### Actual result



### Documentation Changes Required
